frontend/src/components/user/CustomButton.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
```

Comment:
Maybe also rename it LoginButton? As long as this button used only as a login button?

Reply:
it will be used across all forms